ui/src/routes/+page.svelte

```diff
@@ -11,156 +11,102 @@
-			averageTripTime: 0,
-			lostPackets: 0,
-			signal: 0,
-			dataRateTotal: 0,
-			stationStatus: DSStationStatus.Unknown
-		},
-		{
-			teamNumber: 9876,
-			alliance: AllianceType.Blue,
-			station: StationType.Station2,
-			connection: true,
-			dsLinkActive: false,
-			radioLink: false,
-			rioLink: false,
-			linkActive: false,
-			isEnabled: false,
-			isAuto: false,
-			isBypassed: false,
-			isEStopPressed: false,
-			isEStopped: false,
-			battery: 0,
-			monitorStatus: MonitorStatusType.Unknown,
-			averageTripTime: 0,
-			lostPackets: 0,
-			signal: 0,
-			dataRateTotal: 0,
-			stationStatus: DSStationStatus.Unknown
-		},
-		{
-			teamNumber: 4321,
-			alliance: AllianceType.Blue,
-			station: StationType.Station3,
-			connection: true,
-			dsLinkActive: false,
-			radioLink: false,
-			rioLink: false,
-			linkActive: false,
-			isEnabled: false,
-			isAuto: false,
-			isBypassed: false,
-			isEStopPressed: false,
-			isEStopped: false,
-			battery: 0,
-			monitorStatus: MonitorStatusType.Unknown,
-			averageTripTime: 0,
-			lostPackets: 0,
-			signal: 0,
-			dataRateTotal: 0,
-			stationStatus: DSStationStatus.Waiting
-		},
-		{
-			teamNumber: 2468,
-			alliance: AllianceType.Red,
-			station: StationType.Station1,
-			connection: true,
-			dsLinkActive: true,
-			radioLink: false,
-			rioLink: false,
-			linkActive: false,
-			isEnabled: false,
-			isAuto: false,
-			isBypassed: false,
-			isEStopPressed: false,
-			isEStopped: false,
-			battery: 0,
-			monitorStatus: MonitorStatusType.Unknown,
-			averageTripTime: 0,
-			lostPackets: 0,
-			signal: 0,
-			dataRateTotal: 0,
-			stationStatus: DSStationStatus.Unknown
-		},
-		{
-			teamNumber: 1357,
-			alliance: AllianceType.Red,
-			station: StationType.Station2,
-			connection: true,
-			dsLinkActive: true,
-			radioLink: true,
-			rioLink: true,
-			linkActive: false,
-			isEnabled: false,
-			isAuto: false,
-			isBypassed: false,
-			isEStopPressed: false,
-			isEStopped: false,
-			battery: 0,
-			monitorStatus: MonitorStatusType.Unknown,
-			averageTripTime: 0,
-			lostPackets: 0,
-			signal: 0,
-			dataRateTotal: 0,
-			stationStatus: DSStationStatus.Unknown
-		},
-		{
-			teamNumber: 3242,
-			alliance: AllianceType.Red,
-			station: StationType.Station3,
-			connection: true,
-			dsLinkActive: true,
-			radioLink: true,
-			rioLink: true,
-			linkActive: true,
-			isEnabled: false,
-			isAuto: false,
-			isBypassed: false,
-			isEStopPressed: false,
-			isEStopped: false,
-			battery: 0,
-			monitorStatus: MonitorStatusType.Unknown,
-			averageTripTime: 0,
-			lostPackets: 0,
-			signal: 0,
-			dataRateTotal: 0,
-			stationStatus: DSStationStatus.Unknown
-		}
-	];
+	let blue1: FieldMonitorData = Blue1Default;
+	let blue2: FieldMonitorData = Blue2Default;
+	let blue3: FieldMonitorData = Blue3Default;
+	let red1: FieldMonitorData = Red1Default;
+	let red2: FieldMonitorData = Red2Default;
+	let red3: FieldMonitorData = Red3Default;
+
+	let matchNumber = 999;
```

Comment:
Longer term I wonder if we want to use a different number as the fallback since 999 is also the test match number

Reply:
I plan on adding a connection status display, so I think that will take care of "am I looking at real data".